### PR TITLE
livePose log

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

**Comment by fredyshox** [comment]
1184 b -> 312 b

File: cereal/log.capnp [hunk 1]
```diff
@@ -1252,6 +1252,28 @@ struct LiveLocationKalman {
...
+
+struct LivePose {
+  # More info on reference frames:
+  # https://github.com/commaai/openpilot/tree/master/common/transformations
+  orientationNED @0 : Measurement32;
+  velocityDevice @1 : Measurement32;
+  accelerationDevice @2: Measurement32;
+  angularVelocityDevice @3 : Measurement32;
+
+  inputsOK @4 :Bool = true;
```
Comment (adeebshihadeh) [inline cereal/log.capnp:1264]: these defaults made sense for reading old logs, but think we want them to default to False now

File: cereal/log.capnp [hunk 2]
```diff
@@ -1252,6 +1252,28 @@ struct LiveLocationKalman {
...
+  posenetOK @5 :Bool = true;
+  sensorsOK @6 :Bool = true;
+  deviceStable @7 :Bool = true;
```
Comment (adeebshihadeh) [inline cereal/log.capnp:1267]: let's remove if we don't use now?

File: cereal/log.capnp [hunk 3]
```diff
@@ -1252,6 +1252,28 @@ struct LiveLocationKalman {
...
+  filterState @8 : Measurement32;
+
+  struct Measurement32 {
+    value @0 : List(Float32);
```
Comment (adeebshihadeh) [inline cereal/log.capnp:1271]: lists tend to be annoying (checking length, building the list, etc.). can we use xyz here instead?

Comment (fredyshox) [reply cereal/log.capnp:1270]: that complicates a little bit the migration on xx side, but its doable

**Comment by fredyshox** [comment]
This lowers the size of a qlog to 550k with bz2 and 490k with zstd

**Comment by fredyshox** [comment]
> 1208 b -> 312 b

Update: 1208 b -> 216 b

**Pull request merged**